apps/Learning-Buddy/screens/SampleTestScreen.jsx

```diff
@@ -0,0 +1,75 @@
+import { StyleSheet, Text, View, SafeAreaView, ScrollView } from 'react-native';
+import React, { useContext } from 'react';
+import { colors } from '../config/colors';
+import FunctionOnPressBigButton from '../components/FunctionOnPressBigButton';
+import { StyleSheetContext } from '../providers/StyleSheetProvider';
+import SampleTestData from '../data/SampleTestData.json';
+
+export const SampleTestScreen = () => {
+  const styles = useContext(StyleSheetContext);
+
+  function ShuffleShuffle(array1, array2) {
+    // populate with values to shuffle
+    let shuffledArray = [];
+    shuffledArray.push(...array2);
+    shuffledArray.push(array1);
+
+    // shuffle answer array
+    shuffledArray.sort(() => (Math.random() > 0.5 ? 1 : -1));
+
+    // return shuffled array
+    return shuffledArray;
+  }
+
+  return (
+    <SafeAreaView style={localStyles.container}>
+      <ScrollView>
+        <Text style={styles.pageTitle}>SAMPLE TEST</Text>
+        {SampleTestData.questions.map((e, i) => {
+          return (
+            <View style={localStyles.card} key={i}>
+              <Text>Question {i + 1}</Text>
+              <Text>{e.question1}</Text>
+              <Text>Correct Answer: {e.options.Correct}</Text>
+              <Text>Incorrect Answers: {e.options.Incorrect}</Text>
+              <Text>
+                TODO: Radio Buttons:{' '}
+                {ShuffleShuffle(e.options.Correct, e.options.Incorrect)}
+              </Text>
+            </View>
+          );
+        })}
+      </ScrollView>
+      <View style={localStyles.footer}>
+        <FunctionOnPressBigButton
+          content={'Submit'}
+          onPress={() => {
```

Comment:
I'm a bit torn on this; ideally I'd like it if you could find a way to pass along an array with their choices as well as the data you were originally provided with, and then another function could perform the math of evaluating it and sending results to the next screen. The button would also then ideally be disabled if not all questions had been answered (something like, if you had an array with all answers and its length is not equal to the number of questions you're generating, the button is disabled, else it's not disabled... maybe using a useEffect watching the length of that array?)

If for now you could update this button to pass along an array of answers, as well as the original object, to another screen (e.g. "Home Screen" for now, perhaps) that's probably close enough.